packages/fx-core/src/component/resource/appManifest/appStudio.ts

```diff
@@ -747,31 +749,7 @@ export async function updateManifestV3(
     }
   }
 
-  const appStudioTokenRes = await ctx.tokenProvider.m365TokenProvider.getAccessToken({
-    scopes: AppStudioScopes,
-  });
-  if (appStudioTokenRes.isErr()) {
-    return err(appStudioTokenRes.error);
-  }
-  const appStudioToken = appStudioTokenRes.value;
-
   try {
-    const localUpdateTime = (await fs.stat(manifestFileName)).mtime.getTime();
```

Comment:
Remove this logic as file state timestamp cannot be used to distinguish local update time.

E.g.
User does not modify template, the local time will always be the same and less than dev portal update time.

Reply:
In my understanding, localUpdateTime is always updated when building Teams app package. Correct me if I was wrong